core/controllers/story_editor.py

```diff
@@ -228,7 +253,11 @@ class StoryPublishHandler(
 
     @acl_decorators.can_edit_story
     def put(self, story_id: str) -> None:
-        """Published/unpublished given story."""
+        """Published/unpublished given story.
```

Comment:
Please keep the tenses consistent with other handlers.

Reply:
I have changed the tenses.